src/containers/Ecotip/Ecotip.css

```diff
@@ -0,0 +1,14 @@
+.ecotip-card {
+  background-color: #97D8C4;
+  opacity: 0.82;
+  border: 2px solid lightblue;
+  border-radius: 10px;
```

Comment:
There are two border radius rules in this declaration

Reply:
Good catch -- removing now